image_resize_filter.module

```diff
@@ -153,7 +153,9 @@ function image_resize_filter_get_images($settings, $text) {
 
       // Build a list of acceptable language prefixes.
       $lang_codes = '';
-      if (array_key_exists('locale-url', variable_get('language_negotiation_language', array())) && variable_get('locale_language_negotiation_url_part', 0) == 0) {
+      $language_negotiation_language = config_get('language.settings', 'language_negotiation');
+      $language_negotiation_url_part = config_get('locale.settings', 'language_negotiation_url_part');
+      if (array_key_exists('locale-url', $language_negotiation_language) && $language_negotiation_url_part == 'path_prefix') {
```

Comment:
I don't think this condition works as expected, as it's not the array_key, but the value. An example:

```
array (
  'language' => 
  array (
    0 => 'locale-url',
    1 => 'language-default',
  ),
)
```